src/main/java/org/broadinstitute/hellbender/tools/walkers/fasta/ShiftFasta.java

```diff
@@ -0,0 +1,245 @@
+package org.broadinstitute.hellbender.tools.walkers.fasta;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import htsjdk.samtools.reference.FastaReferenceWriter;
+import htsjdk.samtools.reference.FastaReferenceWriterBuilder;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.engine.ReferenceDataSource;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import picard.cmdline.programgroups.ReferenceProgramGroup;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ListIterator;
+
+/**
+ * Create a fasta with the bases shifted by offset
+ *
+ * delta1 = offset - 1
+ * delta2 = total - delta1
+ *
+ * To shift forward:
+ * if you are given a position in the regular fasta (pos_r) and want the position in the shifted fasta (pos_s):
+ * if pos_r > delta1 => pos_s = pos_r - delta1  ==   pos_r - offset +1
+ *   otherwise          pos_s = pos_r + delta2  ==   pos_r + total - offset + 1
+ *
+ * To shift back:
+ * if you are given a position in the shifted fasta (pos_s) and want the position in the regular fasta (pos_r):
+ * if pos_s > delta2 => pos_r = pos_s - delta2  ==   pos_s - total + offset - 1
+ *   otherwise          pos_r = pos_s + delta1  ==   pos_s + offset - 1
+ *
+ * Example command line:
+ * ShiftFasta
+ * -R "testfiles/shift/ecoli/Escherichia_coli_2017C-4173W12.fa"   // the reference to shift
+ * -O "testfiles/shift/ecoli/shifted.fa"                          // the shifted fasta
+ * --shift-back-output "testfiles/shift/ecoli/shiftback.chain"    // the shiftback chain file to use when lifting over
```

Comment:
I would scrub out the folder references here and replace them with something like (<CIRCULAR.FASTA> etc...). Its generally confusing to see references like these in the docs especially when the data aren't checked into our repo.